app-emulation/open-vm-tools/open-vm-tools-12.5.0-r1.ebuild

```diff
@@ -0,0 +1,160 @@
+# Copyright 2007-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools linux-info pam systemd udev
+
+MY_P="${P}-24276846"
+
+DESCRIPTION="Tools for VMware guests"
+HOMEPAGE="https://github.com/vmware/open-vm-tools"
+SRC_URI="https://github.com/vmware/open-vm-tools/releases/download/stable-${PV}/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="X +deploypkg +dnet doc +fuse fuse3 gtkmm +icu multimon pam +resolutionkms +ssl +vgauth"
+REQUIRED_USE="
+	multimon? ( X )
+	vgauth? ( ssl )
+	?? ( fuse fuse3 )
+"
+
+RDEPEND="
+	dev-libs/glib
+	net-libs/libtirpc
+	deploypkg? ( dev-libs/libmspack )
+	fuse? ( sys-fs/fuse:0 )
+	fuse3? ( sys-fs/fuse:3 )
+	pam? ( sys-libs/pam )
+	!pam? ( virtual/libcrypt:= )
+	ssl? ( dev-libs/openssl:= )
+	vgauth? (
+		dev-libs/libxml2
+		dev-libs/xmlsec:=
+	)
+	X? (
+		x11-libs/gdk-pixbuf-xlib
+		x11-libs/gtk+:3
+		x11-libs/libSM
+		x11-libs/libXcomposite
+		x11-libs/libXext
+		x11-libs/libXi
+		x11-libs/libXrandr
+		x11-libs/libXrender
+		x11-libs/libXtst
+		gtkmm? (
+			dev-cpp/gtkmm:3.0
+			dev-libs/libsigc++:2
+		)
+		multimon? ( x11-libs/libXinerama )
+	)
+	dnet? ( dev-libs/libdnet )
+	icu? ( dev-libs/icu:= )
+	resolutionkms? (
+		x11-libs/libdrm[video_cards_vmware]
+		virtual/libudev
+	)"
+DEPEND="${RDEPEND}
+	net-libs/rpcsvc-proto"
+BDEPEND="
+	dev-util/glib-utils
+	virtual/pkgconfig
+	doc? ( app-text/doxygen )"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-12.4.5-Werror.patch
+	"${FILESDIR}"/${PN}-12.4.5-icu.patch
+	"${FILESDIR}"/${PN}-12.4.5-xmlsec1-pc.patch
+)
+
+pkg_setup() {
+	local CONFIG_CHECK="~VMWARE_BALLOON ~VMWARE_PVSCSI ~VMXNET3 ~VMWARE_VMCI ~VMWARE_VMCI_VSOCKETS ~FUSE_FS"
+	use X && CONFIG_CHECK+=" ~DRM_VMWGFX"
+	kernel_is -lt 5 5 || CONFIG_CHECK+=" ~X86_IOPL_IOPERM"
+	linux-info_pkg_setup
+}
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	local myeconfargs=(
+		--disable-glibc-check
+		--disable-tests
+		--without-root-privileges
+		$(use_enable multimon)
+		$(use_with X x)
+		$(use_with X gtk3)
+		$(use_with gtkmm gtkmm3)
+		$(use_enable doc docs)
+		$(use_enable resolutionkms)
+		$(use_enable deploypkg)
+		$(use_with pam)
+		$(use_enable vgauth)
+		$(use_with dnet)
+		$(use_with icu)
+		--with-udev-rules-dir="$(get_udevdir)"/rules.d
+		$(use_with fuse fuse 2)
```

Comment:
just disable old fuse and rely solely on fuse 3, there's no point in this choice

Reply:
Alright, done.